test/graph_rewrites/config_for_grappler_test.cc

```diff
@@ -192,13 +191,10 @@ TEST(GrapplerConfig, RConfig2) {
   ASSERT_OK(BackendManager::SetBackendName("CPU"));
 }
 
-// Though Backend is set via NGRAPH_TF_BACKEND
-// The backend set via RewriterConfig takes affect
+// When Backend is set via NGRAPH_TF_BACKEND it takes effect
+// The backend set via RewriterConfig is ignored
 // since that is the only way of setting backend with grappler
```

Comment:
This `// since that is the only way of setting backend with grappler` can be removed since it's not the only way anymore